nepalingo-web/src/App.tsx

```diff
@@ -1,12 +1,36 @@
 import React from "react";
+import {
+  BrowserRouter as Router,
+  Route,
+  Routes,
+  Navigate,
+} from "react-router-dom";
+import User_auth from "./components/user_auth/User_auth";
+import Home from "./pages/Home/Home";
 import Header from "./components/Header";
+import supabase from "./components/user_auth/supabaseClient";
+
+const App: React.FC = () => {
+  const [isAuthenticated, setIsAuthenticated] = React.useState(false);
+
+  React.useEffect(() => {
```

Comment:
Same for useEffect - import it in line 1

Reply:
done